cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -248,7 +248,7 @@ select[multiple] {
     padding: 5px;
 }
 
-.form-label-header {
+.a-label__heading {
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/58f9e5ba585e8bf927c0a971a4b6252335aaf0b3